lib/suspenders/app_builder.rb

```diff
@@ -422,7 +422,6 @@ def setup_segment
 
     def setup_bundler_audit
       copy_file "bundler_audit.rake", "lib/tasks/bundler_audit.rake"
-      append_file "Rakefile", %{\ntask default: "bundler:audit"\n}
```

Comment:
This appeared to be a bug in suspenders.. it was setting the default rake task to `spec`, then setting it again to `bundler:audit` :-/